languages/fsharp/exercises/concept/basics/Basics.fs

```diff
@@ -0,0 +1,9 @@
+module Basics
+
+// TODO: define the 'expectedMinutesInOven' binding
```

Comment:
Should these be indented under `module Basics` to explain where the bindings go? Or does it not work like that?

Reply:
Nope, it does not work like that. Unless it is a nested module